test/ozaria/site/components/interactive/PageInteractive.spec.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-env jasmine */
+import { mount } from '@vue/test-utils'
+import pageInteractive from 'ozaria/site/components/interactive/PageInteractive/index'
+import factories from 'test/app/factories'
+import interactiveApi from 'ozaria/site/api/interactive'
+import draggableOrderingComponent from 'ozaria/site/components/interactive/PageInteractive/draggableOrdering'
+import inserCodeComponent from 'ozaria/site/components/interactive/PageInteractive/insertCode'
+import draggableStatementCompletionComponent from 'ozaria/site/components/interactive/PageInteractive/draggableStatementCompletion'
+
+const createComponent = (values = {}) => {
+  return mount(pageInteractive, {
+    propsData: values
+  })
+}
+
+const interactive = {
+  interactiveType: 'draggable-ordering',
+  promptText: 'Interactive prompt text',
+  _id: 'interactiveId1'
+}
+
+const introLevel = {
+  _id: 'intro-level-id'
+} // Dummy intro level object
+
+let pageInteractiveWrapper
+
+describe('Interactive Page', () => {
+  beforeEach((done) => {
+    spyOn(interactiveApi, 'getInteractive').and.returnValue(Promise.resolve(interactive))
+    spyOn(interactiveApi, 'getSession').and.returnValue(Promise.resolve({}))
+    me.set(factories.makeUser({ permissions: ['admin'] }).attributes)
```

Comment:
For my own understanding. Doesn't this return a new User? Is there something special about resetting the `me` user?

Reply:
This is required because the interactive page checks the admin permission on `me` user to allow/restrict access to the page.